handlers/public_test.go

```diff
@@ -0,0 +1,87 @@
+package handlers
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/companieshouse/payments.api.ch.gov.uk/config"
+	. "github.com/smartystreets/goconvey/convey"
+	"gopkg.in/jarcoal/httpmock.v1"
+)
+
+func TestUnitCreatePaymentSession(t *testing.T) {
+	cfg, _ := config.Get()
+	reqBody := []byte("{\"redirect_uri\": \"dummy-redirect-uri\",\"resource\": \"http://dummy-resource\",\"state\": \"dummy-state\",\"reference\": \"dummy-reference\"}")
+
+	Convey("Empty Request Body", t, func() {
```

Comment:
Where is the successful unit test? These are all failure scenarios.

Reply:
Unit tests updated